firstvoices/backend/search/utils/query_builder_utils.py

```diff
@@ -247,6 +259,17 @@ def get_valid_boolean(input_val):
         return False
 
 
+def get_valid_has_media(input_val):
```

Comment:
This can use get_valid_boolean above, instead. No need for a media-specific one

Reply:
Ideally all our flags will treat odd values the same way

Reply:
Had a quick chat and have updated this PR to match the functionality of other boolean params and added a ticket to enhance the boolean parameter functionality in the near future (FW-5183).